### PR TITLE
Added recipe for joommfutil

### PR DESCRIPTION
**Pull request by davidcortesortuno**
Utilities for the JOOMMF Python package

### CODE REVIEW

File: recipes/joommfutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7" %}
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: It seems like the newest version is 0.7.3

File: recipes/joommfutil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+package:
+  name: joommfutil
+  version: {{ version }}
+
+source:
+  fn: joommfutil-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/81/0e/b50101da4fc63c7dab5cc97231141c1f41871d740805fbdcf216cc83dae7/joommfutil-0.7.tar.gz
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: Please use the canonical pypi.io link:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

File: recipes/joommfutil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  sha256: a9c92c05b0c0c8af79de38e3f943dc09ca159f21c7ce6e8dcaa7027499be7a9d
+
+build:
+  number: 0
+  script: pip install --no-deps .
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: Please install using the `setup.py`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

File: recipes/joommfutil/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+requirements:
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: This should just be `python` and `setuptools`.

File: recipes/joommfutil/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    - pip
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: It looks like `numpy` is also a requirement.

File: recipes/joommfutil/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - joommfutil
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: Please also import:

```
    - joommfutil.typesystem
```

File: recipes/joommfutil/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+about:
+  home: http://joommf.github.io/
+  license: BSD 3-clause
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: Please add the following to the `about` section:

```
license_file: LICENSE
...
dev_url: https://github.com/joommf/joommfutil
```

File: recipes/joommfutil/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  summary: Utilities for JOOMMF
+  # license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - davidcortesortuno
+    - mb1a15
```
Comment (synapticarbors) [inline recipes/joommfutil/meta.yaml]: @mb1a15, @takluyver @fangohr are you ok being added as co-maintainers of the joommfutil package on conda-forge?

Comment (fangohr) [inline recipes/joommfutil/meta.yaml]: Yes.

Comment (takluyver) [inline recipes/joommfutil/meta.yaml]: 👍 . We're all sitting here together working on packaging.

File: recipes/joommfutil/meta.yaml (hunk 3 above)
Comment (takluyver) [inline recipes/joommfutil/meta.yaml]: I think the `pip` installation is preferable, and I've used this in a number of other recipes. See also discussion on #3034

File: recipes/joommfutil/meta.yaml [hunk 9]
```diff
@@ -1,13 +1,15 @@
...
 package:
   name: joommfutil
   version: {{ version }}
 
 source:
   fn: joommfutil-{{ version }}.tar.gz
-  url: https://files.pythonhosted.org/packages/81/0e/b50101da4fc63c7dab5cc97231141c1f41871d740805fbdcf216cc83dae7/joommfutil-0.7.tar.gz
-  sha256: a9c92c05b0c0c8af79de38e3f943dc09ca159f21c7ce6e8dcaa7027499be7a9d
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: sha256: {{ sha256 }}
```
Comment (takluyver) [inline recipes/joommfutil/meta.yaml]: You've got one `sha256:` too many here.

File: recipes/joommfutil/meta.yaml [hunk 10]
```diff
@@ -17,18 +19,22 @@ requirements:
   build:
     - python
     - pip
+    - numpy
```
Comment (takluyver) [inline recipes/joommfutil/meta.yaml]: I think numpy is needed in the `run` section of the requirements - those are the things that get installed when you install this package.

**Pull request merged**